src/main/java/org/launchcode/walkabout/models/dto/UserJournal.java

```diff
@@ -1,12 +1,24 @@
-package src.main.java.org.launchcode.walkabout.models.dto;
-import java.time.LocalDate;
+package org.launchcode.walkabout.models.dto;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+
 import java.time.LocalDateTime;
 import java.util.Scanner;
 
-public class UserJournal {
+
+public class UserJournal{
+
+    @NotBlank(message = "Please add the journal entry.")
+    @Min(0)
     private String journalEntry;
+
+    @NotBlank(message = "Please add the journal location.")
+    @Min(0)
     private String journalLocation;
-    private LocalDateTime journalDate;
+
+    @NotNull(message = "Please add the journal date.")
```

Comment:
Love that you added a journal date! I think it'll be helpful for users to know how relevant the journal entry will be to them.